src/Chapter1.hs

```diff
@@ -520,7 +520,8 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if (abs x) < (abs y) || (abs x) == (abs y) then x else y
```

Comment:
You can use `<=` operator to check on "less than` and "equal" at once 🙂 
Also, the parenthesises are redundant:
```suggestion
closestToZero x y = if abs x <= abs y then x else y
```